scalafix-core/shared/src/main/scala/scalafix/rule/Rule.scala

```diff
@@ -0,0 +1,191 @@
+package scalafix
+package rule
+
+import scala.meta._
+import scalafix.internal.config.MetaconfigPendingUpstream
+import scalafix.internal.config.ScalafixMetaconfigReaders
+import scalafix.internal.config.ScalafixConfig
+import scalafix.syntax._
+import metaconfig.Conf
+import metaconfig.ConfDecoder
+import metaconfig.Configured
+
+/** A Scalafix Rule is a checker/linter/rewrite/auto-fixer.
```

Comment:
same as above, this `checker/linter/rewrites/whatever` is a bit confusing.

Reply:
I agree, removed.